index.html

```diff
@@ -51,6 +53,238 @@ <h3>LET’S CONNECT</h3>
           />
         </div>
       </section>
+      <!-- WORK SECTION -->
+      <section id="work">
+        <div class="card-works">
+          <img
+            src="./images/work/Tonic.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <div class="project-title">Tonic</div>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <div class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="action-box">
+              <div class="button">
+                <div class="btn-text">See Project</div>
+              </div>
```

Comment:
According to the project requirements, this button should be wrapped inside a `<button>` or `<a>` tag. Kindly fix this.

<img width="840" alt="Screen Shot 2022-05-27 at 17 27 40" src="https://user-images.githubusercontent.com/82426682/170796385-a4dda6d3-7391-41aa-82a4-4458e4214222.png">

Additionally, you need to implement the styling for all the interactions.

This is how they look in the Figma design:

<img width="1001" alt="Screen Shot 2022-05-27 at 17 29 26" src="https://user-images.githubusercontent.com/82426682/170796519-b556008c-33c1-42a6-9e4c-c04d4663514c.png">

I suggest you make some research about the `:hover` and `:active` CSS pseudo-classes, it will help you implement this suggestion.

You should implement these changes in all your buttons. I'm marking just this one for brevity.